driver-core/src/main/com/mongodb/client/model/search/SearchOperator.java

```diff
@@ -463,14 +460,15 @@ static PhraseSearchOperator phrase(final Iterable<? extends SearchPath> paths, f
     }
 
     /**
-     * Returns a {@link SearchOperator} that performs a search using a special characters in the search string that can match any character.
+     * Returns a {@link SearchOperator} that performs a search using a special
+     * characters in the search string that can match any character.
      *
+     * @param path  The indexed field to be searched.
      * @param query The string to search for.
-     * @param path The indexed field to be searched.
      * @return The requested {@link SearchOperator}.
      * @mongodb.atlas.manual atlas-search/wildcard/ wildcard operator
      */
-    static WildcardSearchOperator wildcard(final String query, final SearchPath path) {
+    static WildcardSearchOperator wildcard(final SearchPath path, final String query) {
```

Comment:
Parameter order is inconsistent. I refactored this (in a separate commit) to match at least regex, but this is already inconsistent in the API. I left a comment on the [JIRA for revisiting the API](https://jira.mongodb.org/browse/JAVA-5752?focusedCommentId=7027246&focusedId=7027246&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-7027246).